### PR TITLE
Devc merge

### PR DESCRIPTION
**Pull request by ctaylor451**
Fixes for #134 and #79

### CODE REVIEW

File: main/views/message_views.py [hunk 1]
```diff
@@ -256,3 +263,55 @@ def handle_outbound_email_tracking(sender, event, esp_name, **kwargs):
...
+        context['type'] = "become_do"
+
+        # DO PII
+        do = self.request.user
+        context['do'] = do #FIXME: need?
+
+        context['title'] = "Page DO transition"
+
+        # text box canned message
+        do_shift = "0800 October 9 to 0800 October 16"
```
Comment (kduncklee) [inline main/views/message_views.py]: Add TODO to get this date range from the DO calendar

File: main/views/message_views.py [hunk 2]
```diff
@@ -256,3 +263,55 @@ def handle_outbound_email_tracking(sender, event, esp_name, **kwargs):
...
+        initial['author'] = self.request.user.pk
+        members = None
+        period_id = self.request.GET.get('period')
+        period_format = self.request.GET.get('period_format')
+        page = self.request.GET.get('page')
+
+    def get_queryset(self):
+        """Return the member list."""
+        return Member.objects.filter(
+            Q(member_rank='TM') |
```
Comment (kduncklee) [inline main/views/message_views.py]: can do in=['TM', 'FM', ...

File: main/views/message_views.py [hunk 3]
```diff
@@ -58,7 +59,7 @@ def get_queryset(self):
             if period_format == 'invite':
                 members = Member.active.exclude(
                     participant__period=period_id)
-            else:
+            else: #FIXME: is this else vs elif here for a reason?
```
Comment (kduncklee) [inline main/views/message_views.py]: Agreed, elif sounds better

File: main/views/message_views.py (hunk 2 above)
Comment (ctaylor451) [reply main/views/message_views.py]: Much better

File: main/views/message_views.py (hunk 3 above)
Comment (ctaylor451) [reply main/views/message_views.py]: done

File: main/views/message_views.py (hunk 1 above)
Comment (ctaylor451) [reply main/views/message_views.py]: done

**Pull request merged**